### DISCUSSION TITLE
Text Detection: Grouping bounding boxes together

### DISCUSSION THREAD

**Discussion by leezx337** [PaddlePaddle/PaddleOCR#10292] [Q&A]
Hello all,

May I know where to find the file with the algorithm where the individual bounding boxes are grouped together during text detection? The current text detection module detects an entire block of texts as one bounding box. I'm wondering if i can detect them as individual texts instead :) TYIA!

**Comment by starktynt** [comment]
Hi , yes you can detect them as individual text also , check out craft , does the same

**Reply by leezx337** [reply]
Got it thanks!
